CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ else()
     find_package(slac REQUIRED)
 
     find_package(pugixml REQUIRED)
+    find_package(CURL REQUIRED)
```

Comment:
Didn't we need a recent version?  find_package also supports version requirements.

Reply:
Added the minimum version we need